client/bastionai/utils.py

```diff
@@ -117,8 +117,6 @@ def dataset_from_chunks(chunks: Iterator[Chunk]) -> TensorDataset:
     if any([x is None for x in columns]):
         raise Exception(f"Missing column in data wrapper.")
 
-    return TensorDataset(columns, labels)
```

Comment:
Why do you remove this return statement?????

Reply:
It wasn't removed. I branched when the new API wasn't made.

Reply:
Oh I see, so this is PR will require a merge from master at some point